File: Lessons/02_Arrays_CyclicRotation.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(a, k)
    # write your code in Ruby 2.2
    n = a.length 
    ans = Array.new(n)
    n.times do |i|
      ans[(i + k) % n] = a[i]
    end 
    return ans
end
  

File: Python/02_Arrays_CyclicRotation.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(A, K):
    # write your code in Python 3.6
    n = len(A) 
    ans = [0] * n   
    for i in range(n):
        ans[(i + K) % n] = A[i]
    return ans
# end of func